### PR TITLE
Fixes after latest merges

### PR DESCRIPTION
**Pull request by GermanDZ**
Some fixes to solve conflicts after merging #18 and #13.

### CODE REVIEW

**Review by BuonOmo — COMMENTED** [review]
I'm working on those fixes don't worry :)

File: Jenkinsfile [hunk 1]
```diff
@@ -15,6 +15,7 @@ pipeline {
   environment {
     MINITEST_REPORTERS_REPORTS_DIR = "tmp/test_report"
     KEYCLOAK_VERSION = '11.0.0'
+		KEYCLOAK_CLI_VERSION = '11.0.3'
```
Comment (zachar) [inline Jenkinsfile:18]: margin

**Review by BuonOmo — CHANGES_REQUESTED** [review]
Just in case it wasn't clear, I'm struggling with the rebases, please don't merge that one!

File: Jenkinsfile (hunk 1 above)
Comment (GermanDZ) [reply Jenkinsfile:18]: 😱

**Comment by GermanDZ** [comment]
> Just in case it wasn't clear, I'm struggling with the rebases, please don't merge that one!

Use this PR just as a cue of things that are broken right now.

**Pull request closed without merging**